events.html

```diff
@@ -160,18 +146,41 @@ <h3 class="text-uppercase mb-3">
     </div>
     </div>
     </div>
+    <div class="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay="0.3s">
+        <div class="service-item position-relative overflow-hidden bg-secondary d-flex h-100 p-5 ps-0">
+        <div class="ps-4">
+        <div style="width: auto; height: auto;">
+        <a href="calciworkshop.html">
+        <img id="scificalci" class="img-fluid" src="img\Scientific Calculator.jpeg" alt=""></a>
```

Comment:
this image's height and width isn't aligned with that of others, please fix this